packit_service/worker/handlers/copr.py

```diff
@@ -195,7 +195,12 @@ def run(self):
 
         if self.copr_event.chroot == COPR_SRPM_CHROOT:
             url = get_srpm_build_info_url(self.build.id)
-            self.copr_build_helper.report_status_to_all(
+            report_status = (
+                self.copr_build_helper.report_status_to_all
+                if self.package_config.sync_job_statuses
```

Comment:
```suggestion
                if self.job_config.sync_job_statuses
```
shouldn't we be checking the `job_config` rather, in case user configures this differently for different test jobs?